### PR TITLE
Create vue component to mark page as needing the "classic ui"

### PR DESCRIPTION
**Pull request by bdemers**
To use this tag add `<RequireClassicUI/>` at the top of the page.

Write a message for this pull request. The first block
of text is the title and the rest is the description.

image

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/global-components/RequireClassicUI.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <div>
+    <blockquote>
+      <details class="custom-block details">
+        <summary>The <strong>Classic UI</strong> is required for this guide.</summary>
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/global-components/RequireClassicUI.vue:5]: <summary>The Admin Console (Classic UI) is required for this guide.</summary>

File: packages/@okta/vuepress-theme-prose/global-components/RequireClassicUI.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <p>
+          The top left of your Okta Admin Console should say <strong>Classic UI</strong>, if it says <strong>Developer Console</strong> click the dropdown to change it to <strong>Classic UI</strong>.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/global-components/RequireClassicUI.vue:7]: The upper-left corner of the page should say <strong>Classic UI</strong>. If it says <strong>Developer Console</strong>, click the drop-down box to change it to <strong>Classic UI</strong>.

**Comment by bdemers** [comment]
image
@susanharper-okta how does this look?

**Comment by susanharper-okta** [comment]
> image
> @susanharper-okta how does this look?

Looks great. 🚢

**Pull request merged**